shared/devices/container.js

```diff
@@ -40,6 +42,10 @@ type OwnProps = void
 
 function mergeProps(stateProps, dispatchProps, ownProps: OwnProps) {
   const [revoked, normal] = splitAndSortDevices(stateProps._deviceMap)
+  const revokedItems = revoked.map(deviceToItem)
+  const newlyRevokedIds = I.Set(revokedItems.map(d => d.key)).intersect(
+    I.Set(stateProps.newlyChangedItemIds || [])
```

Comment:
now you don't need to wrap this as its a I.Set already

Reply:
👍